src/lib/parquet/src/Flow/Parquet/ParquetFile/Codec.php

```diff
@@ -26,6 +26,7 @@ public function compress(string $data, Compressions $compression) : string
             Compressions::SNAPPY => \snappy_compress($data),
             /** @phpstan-ignore-next-line */
             Compressions::GZIP => \gzencode($data, $this->options->get(Option::GZIP_COMPRESSION_LEVEL)),
+            Compressions::ZSTD => \zstd_compress($data, $this->options->get(Option::ZSTD_COMPRESSION_LEVEL)),
```

Comment:
may I add an int typecast here to avoid the static analysis error?

Reply:
I believe there should be an `getInt` getter, and if not I would add one that would return ints so we wont need to cast anything anywhere :)